woocommerce/woocommerce-gutenberg-products-block
# Critical flow: Can see correct tax

This PR ads an e2e test to check that the right tax is being displayed, both in Cart and Checkout blocks. It also builds on the WC e2e-utils `shopper` functions and adds some useful ones related to blocks.


Fixes #5772 and #5754 






### Testing

#### Automated Tests

### Manual Testing

How to test the changes in this Pull Request:

1. `npm run wp-env`
2. Either run `npm run test:e2e -- Tax` 
3. Make sure all tests pass
4. Make sure e2e tests pass in CI
